docs/PORTABLE_SERVICES.md

```diff
@@ -335,6 +335,27 @@ service data may be placed on the host file system. Use `StateDirectory=` in
 the unit files to enable such behaviour and add a local data directory to the
 services copied onto the host.
 
+## Logging
+
+Several fields are autotmatically added to log messages generated by a portable
+service (or about a portable service, e.g.: start/stop logs from systemd).
+The `PORTABLE=` field will refer to the name of the portable image where the unit
+was loaded from. In case extensions are used, additionally there will be a
+`PORTABLE_ROOT=` field, referring to the name of image used as the base layer
+(i.e.: `RootImage=` or `RootDirectory=`), and one `PORTABLE_EXTENSION=` field per
+each extension image used.
+
+For example, a portable service `app0` using two extensions `app0.raw` and
+`app1.raw`, and a base layer `base.raw`, will create log entries with the
+following fields:
+
+```
+PORTABLE=app0.raw
+PORTABLE_ROOT=base.raw
+PORTABLE_EXTENSION=app0.raw
+PORTABLE_EXTENSION=app1.raw
```

Comment:
I'm also worried about the non-uniqueness of this name that @poettering raised earlier.
There are two general types of fields: app-specific, where short names are OK because it is only the app that needs to coordinate with itself, and system-wide, which are appended to many or a large fraction of log messages. For those, it is important to have namaspacing so that conflicts don't occur, and that it's clear what the field is for.

"PORTABLE_" works as a kind of a namespace, but it's also a normal word, so the namespacing is "weak". ~~What about changing that to "SYSTEMD_PORTABLE_"? Then the chances of misunderstanding or conflict go down to negligible.~~ I guess it's good enough. Considering that you also want to have envvars with the same contents, it makes sense to use the same names.

Reply:
Yeah we were already using PORTABLE*, so just went on with the same prefix. We have been using "portable" for this feature for a while, and these are added only when portabled is used, so the namespacing should be ok, it is not added to generic services, only those attached by portabled.